pre-commit/run-org-hooks

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+"""
+Hook script to run org hooks.
+
+Modified from: https://stackoverflow.com/a/59055569
+"""
+
+import os
+import shutil
+import sys
+from pathlib import Path
+
+
+REPO_HOOK_CONFIGS_DIRNAME = '.hook_configs'
+
+orghooks_dir = os.path.dirname(os.path.realpath(__file__))
+
+
+def copy_configs(dest_dir):
+    """ Make a copy of configurations for hooks if one doesn't exist. """
+    dev_tools_dir = Path(orghooks_dir).parents[0]
+    lint_configs_dir = dev_tools_dir / 'lint-configs'
+    repo_hook_configs_dir = Path(dest_dir) / REPO_HOOK_CONFIGS_DIRNAME
+    shutil.copytree(lint_configs_dir, repo_hook_configs_dir, dirs_exist_ok=True)
+
+
+def main():
+    """ Main entrypoint. """
+    repo_dir = os.getcwd()
+    copy_configs(repo_dir)
+    cfg = os.path.join(orghooks_dir, 'orghooks.yaml')
+    cmd = ['pre-commit', 'run', '--config', cfg, '--files'] + sys.argv[1:]
```

Comment:
would `sys.argv[1:]` be the list of updated files passed by `pre-commit`?

Reply:
yep! you can see the full context [here](https://stackoverflow.com/a/59055569) but it's basically mimicking an include by passing the files from the top-level pre-commit command